app/decorators/address_verification_decorator.rb

```diff
@@ -6,17 +6,23 @@ class AddressVerificationDecorator < ResourceDecorator
   delegate :country, to: :identity, prefix: true, allow_nil: true
 
   def identity_link
-    return if identity.nil?
+    return if model.nil? || identity.nil?
```

Comment:
model can't be nil. decorator should not decorate nil value. it will be resolved by my upper comment

same below for this file